lib/galaxy/workflow/refactor/schema.py

```diff
@@ -211,6 +211,10 @@ class UpgradeToolAction(BaseAction):
     tool_version: Optional[str]
 
 
+class UpgradeWorkflowAction(BaseAction):
+    action_type: Literal['upgrade_workflow']
```

Comment:
What about ``upgrade_all_steps``? I don't know, what you have here is a fine choice I'm just worried it is a little confusing with ``upgrade_subworkflow``.

If no one else feels strongly though, I'm okay with keeping it like it like you have it here I think.

Reply:
Sure, I agree that's clearer.